gui-apps/waybar/waybar-0.9.2.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="Highly customizable Wayland bar for Sway and Wlroots based compositors."
+HOMEPAGE="https://github.com/Alexays/Waybar"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Alexays/${PN^}.git"
+else
+	SRC_URI="https://github.com/Alexays/${PN^}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="network udev pulseaudio systemd tray mpd gtk-layer-shell man-pages"
+
+BDEPEND="
+	virtual/pkgconfig
+	dev-libs/wayland-protocols
+	man-pages? ( >=app-text/scdoc-1.9.2 )
+"
+
+DEPEND="
+	>=dev-cpp/gtkmm-3.22.0:=
+	dev-libs/jsoncpp:=
+	dev-libs/libsigc++:2
+	>=dev-libs/libfmt-5.3.0:=
+	dev-libs/wayland
+	>=dev-libs/spdlog-1.3.1:=
```

Comment:
Also please sort this list. `dev-libs/libfmt < dev-libs/libsigc++`, `dev-libs/spdlog < dev-libs/wayland`.